Source/WebKit/UIProcess/API/Cocoa/WKUIDelegate.h

```diff
@@ -57,6 +57,18 @@ typedef NS_ENUM(NSInteger, WKMediaCaptureType) {
     WKMediaCaptureTypeCameraAndMicrophone,
 } WK_API_AVAILABLE(macos(12.0), ios(15.0));
 
+/*! @enum WKDialogResult
+@abstract Constants returned by showLockdownModeFirstUseMessage to indicate how WebKit should treat first use.
+@constant WKDialogResultShowDefault Indicates that the client did not display a first use message. WebKit should show the default.
+@constant WKDialogResultAskAgain Indicates the client handled the message, but want's to be checked if other WKWebViews are used.
+@constant WKDialogResultHandled Indicates the client handled the message and no further checks are needed.
+*/
+typedef NS_ENUM(NSInteger, WKDialogResult) {
```

Comment:
This was feedback that James gave during the API review process. I'd prefer to stick with this naming, since it's in the official API review submission, until a final approval is made. Feel free to pile on the API review discussion if you want the naming to be reconsidered (I'll e-mail you a link).